src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,31 @@
-import React from 'react';
+import { Post } from '../../types/Post';
+import { PostInfo } from '../PostInfo';
+import { Comment } from '../../types/Comment';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
-);
+export type PostListProps = {
+  posts: Post[],
+  comments: Comment[];
+};
+
+export const PostList: React.FC<PostListProps> = ({ posts, comments }) => {
+  return (
+    <div className="PostList">
+      {posts.map(
+        post => {
```

Comment:
There shouldn't be a need for this, correct comments should be attached to posts in the App.tsx and here only mapping of PostInfo should be happening

Reply:
I added comments to post.